apps/nmreplay/nmreplay.c

```diff
@@ -972,7 +972,7 @@ usage(void)
 {
 	fprintf(stderr,
 	    "usage: nmreplay [-v] [-D delay] [-B {[constant,]bps|ether,bps|real,speedup}] [-L loss]\n"
-	    "\t[-b burst] -f pcap-file -i ifb\n");
+	    "\t[-b burst] -f pcap-file -i <netmap|valeSSS>:<iface_name>\n");
```

Comment:
In this way it's very unclear. Maybe something like
`netmap:ifname|valeSSS:PPP|...

Reply:
Updated!